source/components/executer/exconvrt.c

```diff
@@ -501,8 +504,17 @@ AcpiExConvertToAscii (
         {
             /* Get one hex digit, most significant digits first */
 
-            String[k] = (UINT8)
+            HexChar = (UINT8)
                 AcpiUtHexToAsciiChar (Integer, ACPI_MUL_4 (j));
+
+            /* Supress leading zeros until the first non-zero character */
+
+            if (HexChar == ACPI_ASCII_ZERO && SupressZeros) {
```

Comment:
Could you just move the opening brace to the next blank line in order to be compatible with the ACPICA coding style, thanks!

Reply:
Done